homeassistant/components/rtsp_to_webrtc/strings.json

```diff
@@ -24,6 +24,12 @@
       "server_unreachable": "[%key:component::rtsp_to_webrtc::config::error::server_unreachable%]"
     }
   },
+  "issues": {
+    "deprecated": {
+      "title": "The RTSPtoWebRTC integration is deprecated",
+      "description": "The RTSPtoWebRTC integration is deprecated and will be removed. Please use the {go2rtc} integration instead, which provides a better expercience."
```

Comment:
```suggestion
      "description": "The RTSPtoWebRTC integration is deprecated and will be removed. Please use the {go2rtc} integration instead, which provides a better experience."
```

Reply:
What do you think about mentioning that go2rtc is also already enabled by default? "Please use the {go2rtc} integration instead, which is enabled by default and provides a better experience" -- or telling them explicitly to just remove this integration and it will start working.

Reply:
The part that i was thinking about here is "Please use the {go2rtc} integration instead" -- but how does someone use it since its not available in the UI and what steps they should take? I was thinking that telling them that they don't need to do anything to use it, besides removing the integration, is the important part. I don't remember what the other issue says.

Reply:
Updated the text. Do you still want to add something?